doc/source/release/1.7.0-notes.rst

```diff
@@ -342,7 +342,7 @@ There are two changes to the default behavior of `scipy.stats.mannwhitenyu`:
 Support has been added for PEP 621 (project metadata in ``pyproject.toml``)
 
 We now support a Gitpod environment to reduce the barrier to entry for SciPy
-development; for more details see :ref:`quickstart-gitpod`.
+development; for more details see ``quickstart-gitpod``.
```

Comment:
Should we add a permalink to the 1.7.0 docs?

Reply:
(I was curious and had a look at the view stats for the page, we had 120 visitors on the gitpod page in the last 12 months for 7.7M page views. And 5.2K visitors on the release page. So not sure it would be really visible or make a big impact.)

If you want to make this visible, we could consider a link to this old page but directly in the `dev_quickstart`. It would be a mention saying that GitPod is another option and we used to have docs about that, see ...

This would address your other suggestion. What do you think??

Reply:
Sounds good!

Reply:
Done